### ISSUE TITLE
[Issue]: kids search finds porn stars

### ISSUE THREAD

**Issue by cevznriny** [jellyfin/jellyfin#8422]
### Please describe your bug

A search for "Barbie" in my Kids profile finds a porn star called "Barbie Sins".
My xxx library is not accessible from my Kids profile.
I really don't want my kids watching porn.

Actors access really should be filtered by the library they come from.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nielsvanvelzen** [comment]
I believe this issue is fixed in #8399 and will be released in 10.8.5.

**Comment by cevznriny** [comment]
> I believe this issue is fixed in #8399 and will be released in 10.8.5.

Thanks. 
Do you know if there is a quick fix? 
Can I purge all the Actors from the database?

**Comment by crobibero** [comment]
Fixed in 10.8.5

**Comment by Marce1ek** [comment]
> Fixed in 10.8.5

Still have similar problem in 10.8.7
These korean actors are not in the marvel movies😨
![QQ截图20221129001353](https://user-images.githubusercontent.com/99329516/204327457-d2581dc4-aead-4a42-b221-e8b6bd70e94f.jpg)
![QQ截图20221129001116](https://user-images.githubusercontent.com/99329516/204326677-a66abd29-958c-44cf-af21-4799de3132a1.jpg)
